src/Arcus.Testing.Logging.MSTest/Extensions/ILoggerBuilderExtensions.cs

```diff
@@ -22,7 +22,9 @@ public static ILoggingBuilder AddMSTestLogging(this ILoggingBuilder builder, Tes
             ArgumentNullException.ThrowIfNull(builder);
             ArgumentNullException.ThrowIfNull(testContext);
```

Comment:
I cannot comment on the correct line, but I saw that xUnit is mentioned in the XML comment on L15.  Since we're in the MsTest namespace, I believe that this is not correct :)

Reply:
And on L17, NUnit is mentioned.

Reply:
Fixed in https://github.com/arcus-azure/arcus.testing/pull/454/commits/855d3ed5a2f6efb5d0bf75bb500f41ab6b979e6f